jekyll/_cci2/code-coverage.md

```diff
@@ -308,8 +559,54 @@ a tool called phpdbg; you can generate a report using the command `phpdbg -qrr v
 In the following basic `.circleci/config.yml` we upload the coverage reports in
 the `store_artifacts` step at the end of the config.
 
+{:.tab.php_example.Cloud}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/php:8.1-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run:
+          name: "Run tests"
+          command: phpdbg -qrr vendor/bin/phpunit --coverage-html build/coverage-report
+      - store_artifacts:
+          path:  build/coverage-report
+```
+
+{:.tab.php_example.Server_3}
+```yaml
+version: 2.1
+orbs:
+  browser-tools: circleci/browser-tools@1.2.3
+jobs:
+  build:
+    docker:
+      - image: cimg/php:8.1-browsers
+        auth:
+          username: mydockerhub-user
+          password: $DOCKERHUB_PASSWORD  # context / project UI env-var reference
+    steps:
+      - checkout
+      - run:
```

Comment:
If we're going to use a browser testing example and include the orb, we need to run the orb. Something here like:

`- run: browser-tools/install-browser-tools`